internal/static/integrations/gcp_exporter/self_pruning_store.go

```diff
@@ -0,0 +1,110 @@
+package gcp_exporter
+
+import (
+	"context"
+	"sync"
+	"time"
+
+	"github.com/go-kit/log"
+	"github.com/go-kit/log/level"
+	"github.com/prometheus-community/stackdriver_exporter/collectors"
+	"github.com/tilinna/clock"
+	"go.uber.org/atomic"
+	"google.golang.org/api/monitoring/v3"
+)
+
+// Five minutes in seconds for Unix comparisons
+const DefaultPruneWindow int64 = 5 * 60 * 60
```

Comment:
Is this right? Wouldn't this be 300 minutes?

Reply:
😆 🤦 yeah you're right

Reply:
I refactored the tests + pruning window implementation so that the tests would fail from this problem. Pruning window is hard coded now, originally I was going to try to use it to get around clock mocking but found the clock implementation used elsewhere already.